FHEM/14_SD_WS.pm

```diff
@@ -774,54 +873,86 @@ sub SD_WS_WH2SHIFT($){
   <b>Known models:</b>
   <ul>
     <li>Bresser 7009994</li>
     <li>BresserTemeo</li>
     <li>Conrad S522</li>
-	<li>Opus XT300</li>
+		<li>Opus XT300</li>
     <li>PV-8644 infactory Poolthermometer</li>
     <li>Renkforce E0001PA</li>
-	<li>WH2 (TFA Dostmann/Wertheim 30.3157(Temperature only!) (sold in Germany), Agimex Rosenborg 66796 (sold in Denmark),ClimeMET CM9088 (Sold in UK)</li>
-  </ul>
-  <br>
-  New received device are add in fhem with autocreate.
-  <br><br>
+		<li>WH2 (TFA Dostmann/Wertheim 30.3157 (sold in Germany), Agimex Rosenborg 66796 (sold in Denmark),ClimeMET CM9088 (Sold in UK)</li>
+		<li>Weatherstation Auriol IAN 283582 Version 06/2017 (Lidl), Modell-Nr.: HG02832D</li>
+		<li>Weatherstation TFA 35.1140.01 with temperature, humidity and windspeed sensor TFA 30.3222.02</li>
+  </ul><br><br>
 
   <a name="SD_WS_Define"></a>
-  <b>Define</b> 
-  <ul>The received devices created automatically.<br>
-  The ID of the defice is the cannel or, if the longid attribute is specified, it is a combination of channel and some random generated bits at powering the sensor and the channel.<br>
-  If you want to use more sensors, than channels available, you can use the longid option to differentiate them.
-  </ul>
-  <br>
+  <b>Define</b><br><br>
+  <ul>
+		Newly received sensors are usually automatically created in FHEM via autocreate.<br>
+		It is also possible to set up the devices manually with the following command:<br><br>
+    <code>define &lt;name&gt; SD_WS &lt;code&gt; </code> <br><br>
+    &lt;code&gt; is the channel or individual identifier used to identify the sensor.<br>
+  </ul><br><br>
+
   <a name="SD_WS Events"></a>
-  <b>Generated readings:</b>
-  <br>Some devices may not support all readings, so they will not be presented<br>
+  <b>Generated readings:</b><br><br>
   <ul>
-     <li>battery: (low or ok)</li>
-     <li>channel: (The Channelnumber (number if)</li>
-	 <li>humidity: (The humidity (1-100 if available)</li>
-	 <li>sendmode (automatic or manual)</li>
-	 <li>state (T: H:)</li>
-     <li>temperature (&deg;C)</li>
-  </ul>
-  <br>
-  <b>Attributes</b>
+		Some devices may not support all readings, so they will not be presented<br>
+	</ul>
   <ul>
-    <li><a href="#do_not_notify">do_not_notify</a></li>
-    <li><a href="#ignore">ignore</a></li>
-    <li>model (other / E0001PA / S522)<br>
-	The sensors of the "SD_WS_33 series" have different battery bit positions. If the battery bit is detected incorrectly (low instead of ok), then you can adjust with the model selection of the sensor.<br>
-	All sensors except the E0001PA and S522 model are created after an autocreate as model "other". If you own a Conrad S522 or Renkforce E0001PA sensor, set the model for proper battery bit detection.</li>
-    <li><a href="#showtime">showtime</a></li>
-    <li><a href="#readingFnAttributes">readingFnAttributes</a></li>
-  </ul><br>
-
-  <b>Set</b> <ul>N/A</ul><br>
+    <li>battery: (low or ok)</li>
```

Comment:
Das heisst jetzt nur noch batteryState oder?

Reply:
Ja, laut https://wiki.fhem.de/wiki/DevelopmentGuidelines#BatteryReadings gibt es nur noch diese:
    batteryState
    batteryPercent
    batteryVoltage

Reply:
Mit meiner Frage wollte ich darauf hinweisen, dass in der commandref battery und nicht batterystate steht.